smallrye-reactive-messaging-provider/pom.xml

```diff
@@ -53,6 +53,18 @@
       <scope>test</scope>
     </dependency>
 
+    <dependency>
+      <groupId>io.smallrye.testing</groupId>
+      <artifactId>smallrye-testing-utilities</artifactId>
+      <scope>test</scope>
+    </dependency>
+
+    <dependency>
+      <groupId>org.jboss.logmanager</groupId>
+      <artifactId>jboss-logmanager</artifactId>
+      <version>2.1.17.Final</version>
```

Comment:
Should we declare this version in the root pom? Maybe even the dependency in the dependency management?

Reply:
Done